sklearn/svm/base.py

```diff
@@ -181,8 +181,11 @@ def fit(self, X, y, sample_weight=None):
         # In binary case, we need to flip the sign of coef, intercept and
         # decision function. Use self._intercept_ internally.
```

Comment:
Please update the comment to tell "Use self._intercept_ and self._dual_coef_ internally".